static/page-templates/page-templates.json

```diff
@@ -2,84 +2,94 @@
 	"description": "WordPress.com page templates",
 	"templates": [
 		{
-			"title": "rockfield",
-			"defaults": "rockfield",
-			"vertical-whitelist": []
-		},
-		{
-			"title": "rockfield2",
-			"defaults": "rockfield",
-			"vertical-whitelist": []
-		},
-		{
-			"title": "leven",
-			"defaults": "leven",
-			"vertical-whitelist": []
-		},
-		{
-			"title": "alves",
-			"defaults": "alves",
-			"vertical-whitelist": []
-		},
-		{
-			"title": "alves2",
-			"defaults": "alves",
-			"vertical-whitelist": []
-		},
-		{
-			"title": "balasana",
-			"defaults": "balasana",
-			"vertical-whitelist": []
-		},
-		{
-			"title": "coutoire",
-			"defaults": "coutoire",
-			"vertical-whitelist": []
-		},
-		{
-			"title": "exford",
-			"defaults": "exford",
-			"vertical-whitelist": []
-		},
-		{
-			"title": "rockfield",
-			"defaults": "rockfield",
-			"vertical-whitelist": []
-		},
-		{
-			"title": "stratford",
-			"defaults": "stratford",
-			"vertical-whitelist": []
-		},
-		{
-			"title": "stratford2",
-			"defaults": "stratford",
-			"vertical-whitelist": []
-		},
-		{
-			"title": "mayland",
-			"defaults": "mayland",
-			"vertical-whitelist": []
-		},
-		{
-			"title": "mayland2",
-			"defaults": "mayland",
-			"vertical-whitelist": []
-		},
-		{
-			"title": "maywood",
-			"defaults": "maywood",
-			"vertical-whitelist": []
-		},
-		{
-			"title": "maywood2",
-			"defaults": "maywood",
-			"vertical-whitelist": []
-		},
-		{
-			"title": "rivington",
-			"defaults": "rivington",
-			"vertical-whitelist": []
+			"title": "Vesta",
+			"slug": "vesta",
+			"template": "mayland2",
+			"theme": "mayland",
+			"src": "https://public-api.wordpress.com/rest/v1/template/demo/mayland/mayland2/",
```

Comment:
I don't think we actually need these URLs, as they can be built the `theme` and `template` fields. We're doing that already here: https://github.com/Automattic/wp-calypso/blob/6f56c69b594669f7c1aa83af7770a69fdd14ed8a/client/landing/gutenboarding/onboarding-block/style-preview/preview.tsx#L40-L43

(Can be handled in a follow-up tho.)

Reply:
you are totally right there. We can build the entire thing from the rest of the fields !